packages/core/test/lib/commands/unbox.js

```diff
@@ -43,30 +43,28 @@ describe("commands/unbox.js", () => {
         const promises = [];
         for (const path of invalidBoxFormats) {
           promises.push(
-            new Promise(resolve => {
-              const callback = error => {
-                error ? assert(true) : assert(false);
-                resolve();
-              };
-              unbox.run({ _: [`${path}`] }, callback);
-            })
+            unbox
+              .run({_: [`${path}`]})
+              .then(() => assert.fail())
+              .catch(_error => assert(true))
           );
         }
         return Promise.all(promises);
       });
     });
 
     describe("successful unboxes", () => {
-      it("runs when passed valid box input", done => {
+      it("runs when passed valid box input", async () => {
```

Comment:
[Mocha says](https://mochajs.org/#arrow-functions) to use `function () {}` not `() => {}`:

```suggestion
      it("runs when passed valid box input", async function () {
```